### PR TITLE
fix: only log slow sql query parameters if trace logging is enabled

### PR DESCRIPTION
**Pull request by mezgerj**
This corrects https://github.com/k3s-io/kine/issues/305 by only dumping the sql query parameters in the slow sql log if the log level is set to trace.

### CODE REVIEW

**Review by brandond — APPROVED** [review]

**Comment by brandond** [comment]
lgtm, thanks!

**Comment by mezgerj** [comment]
@brandond thanks for the review. What is the process to get this merged?

**Review by vitorsavian — APPROVED** [review]

**Comment by vitorsavian** [comment]
hey @mezgerj, the process of merging is that any PR need two approvals, after the two approvals one of the devs can go and merge. 

I'll merge your PR, tks for the contribution!!

**Pull request merged**
